edx_proctoring/views.py

```diff
@@ -241,12 +242,30 @@ def get(self, request, course_id, content_id):
                 # prerequisites for practice or onboarding exams
                 elif exam['is_proctored'] and not exam['is_practice_exam']:
                     exam = check_prerequisites(exam, request.user.id)
+
+                # if user hasn't completed required onboarding exam before taking
+                # proctored exam we need to navigate them to it with a link
+                if (exam['is_proctored'] and attempt_data and
+                        attempt_data['attempt_status'] in ProctoredExamStudentAttemptStatus.onboarding_errors):
+                    onboarding_exam = ProctoredExam.objects.filter(
```

Comment:
course teams may have set up staff only exams for testing or the onboarding exam may be inaccessible for some other reason. This is handled properly in StudentOnboardingStatusView.

However, this looks like an existing bug for the onboarding_error.html view as it is. Can we just file a separate bug ticket to keep track.

Reply:
https://openedx.atlassian.net/browse/EDUCATOR-5828

Reply:
^ awesome thanks, left to comment on that since I think fixing his low priority for now